ledger-core/network/nds/uniproto/l2/uniserver/server.go

```diff
@@ -51,6 +51,7 @@ func NewUnifiedServer(dispatcher uniproto.Dispatcher, logger MiniLogger) *Unifie
 		panic(throw.IllegalValue())
 	}
 	s := &UnifiedServer{logger: logger}
+	s.transportProvider = defaultProvider
```

Comment:
its not good idea to use global object

Reply:
What the best way for this?

Reply:
s.transportProvider = &DefaultTransportProvider{}

Reply:
Fixed